db/migrate/20220310110609_add_rejection_reasons_type_to_application_choices.rb

```diff
@@ -0,0 +1,9 @@
+class AddRejectionReasonsTypeToApplicationChoices < ActiveRecord::Migration[6.1]
+  def up
+    add_column :application_choices, :rejection_reasons_type, :string, null: false, default: 'reasons_for_rejection'
```

Comment:
We currently have 2 types of reasons for rejection: structured and free text, do we want to differentiate those?

Reply:
Want to send this one to BigQuery too @claire-hughez?